skycoin/skywire
# Adding trusted visors from config on startup

Did you run `make format && make check`? yes

Fixes #368

 Changes:	
- Add trusted visors from config on startup

How to test this PR:
- Set branch of `skywire-services` to https://github.com/SkycoinPro/skywire-services/pull/184
- Run generic integration env
- Wait 5 seconds
- Run `./bin/skywire-cli --rpc $RPC_A visor ls-tp`
- Run `./bin/skywire-cli --rpc $RPC_C visor ls-tp`
- Run `./bin/skywire-cli --rpc $RPC_B visor ls-tp`
- There should be 1 transport on `$RPC_A` and `$RPC_C`, 2 transports on `$RPC_B`; all transports should be up
